Samples/2.0/AzureWorkerRoleSample/OrleansClient/OrleansClient.csproj

```diff
@@ -0,0 +1,262 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could this be converted to the new project format?

Reply:
I tried creating a new project and adding the necessary packages, but I wasn't able to add it as a Worker Role...